src/Components/People.js

```diff
@@ -0,0 +1,60 @@
+import { Component } from 'react';
+import axios from 'axios';
+
+class People extends Component {
+    state = {
+        searchPerson: '',
+        person: {},
+        showError: false
+    }
+
+    handleChange = (e) => {
+        this.setState({
+            searchPerson: e.target.value
+        })
+    }
+
+    handleSubmit = async (e) => {
+        e.preventDefault()
+        const { searchPerson } = this.state
+        const { data } = await axios.get(
+            `https://ghibliapi.herokuapp.com/people?name=${searchPerson}`
+        )
+        console.log(data)
+        this.setState({
+            searchPerson: '',
+            person: data,
+            showError: false
+        })
+
+        if (data.length === 0) {
+            this.setState({
+                searchPerson: '',
+                person: {},
+                showError: true
+            })
+        }
+    }
+
+    render() {
+        const { searchPerson, person, showError } = this.state
+        return (
+            <div>
+                <h1>Search for a Person</h1>
+                <form onSubmit={this.handleSubmit}>
+                    <input onChange={this.handleChange} value={searchPerson} type='text' placeholder='Find Your Person' />
+                    <button type='submit'>Submit</button>
+                </form>
+                {showError ? (
+                    <h1>Not Found</h1>) : person[0] ? (
```

Comment:
I'm not really a fan of nested ternary statements. They're kind of a pain to read! This is my personal preference rather than a rule to live by.

Here's one way to do that:
One line:
`{showError && <h1>Not Found</h1>}`
Another line:
`{person[0] && [that whole div with person's info]}`